newIDE/app/src/ProjectManager/index.js

```diff
@@ -270,6 +275,7 @@ type State = {|
   projectPropertiesDialogOpen: boolean,
   variablesEditorOpen: boolean,
   extensionsSearchDialogOpen: boolean,
+  name: any,
```

Comment:
That's a super generic name for a variable :/ 
Can you think of something more descriptive? Also never use any for types.

Reply:
ok. what of `currentLayoutName`?